website/docs/r/apigatewayv2_authorizer.html.markdown

```diff
@@ -47,17 +47,24 @@ The following arguments are supported:
 
 * `api_id` - (Required) The API identifier.
 * `authorizer_type` - (Required) The authorizer type. Valid values: `JWT`, `REQUEST`.
-For WebSocket APIs, specify `REQUEST` for a Lambda function using incoming request parameters.
- For HTTP APIs, specify `JWT` to use JSON Web Tokens.
+Specify `REQUEST` for a Lambda function using incoming request parameters.
+For HTTP APIs, specify `JWT` to use JSON Web Tokens.
 * `identity_sources` - (Required) The identity sources for which authorization is requested.
```

Comment:
I think identify_souces should be optional - https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-lambda-authorizer.html

Reply:
@account-colind Yes, you are correct, `identity_sources` [is optional](https://docs.aws.amazon.com/apigatewayv2/latest/api-reference/apis-apiid-authorizers-authorizerid.html#apis-apiid-authorizers-authorizerid-properties). I'll correct that.
Thanks.

Reply:
Be aware

https://github.com/terraform-providers/terraform-provider-aws/blob/1366d440e836c555b05072e69f5a69a136bb7208/vendor/github.com/aws/aws-sdk-go/service/apigatewayv2/api.go#L7536-L7538

```console
$ make testacc TEST=./aws TESTARGS='-run=TestAccAWSAPIGatewayV2Authorizer_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSAPIGatewayV2Authorizer_basic -timeout 120m
=== RUN   TestAccAWSAPIGatewayV2Authorizer_basic
=== PAUSE TestAccAWSAPIGatewayV2Authorizer_basic
=== CONT  TestAccAWSAPIGatewayV2Authorizer_basic
    resource_aws_apigatewayv2_authorizer_test.go:22: Step 1/2 error: terraform failed: exit status 1
        
        stderr:
        
        Error: error creating API Gateway v2 authorizer: InvalidParameter: 1 validation error(s) found.
        - missing required field, CreateAuthorizerInput.IdentitySource.
        
        
        
--- FAIL: TestAccAWSAPIGatewayV2Authorizer_basic (25.65s)
FAIL
FAIL	github.com/terraform-providers/terraform-provider-aws/aws	25.694s
FAIL
GNUmakefile:27: recipe for target 'testacc' failed
make: *** [testacc] Error 1
```

Reply:
Fixed.